src/bci_build/package/apache2/README.md.j2

```diff
@@ -0,0 +1,99 @@
+# {{ image.pretty_name }} Container Image
+
+The Apache HTTP Server Project is an effort to develop and maintain an open-source HTTP server for modern operating systems including UNIX and Windows. The goal of this project is to provide a secure, efficient and extensible server that provides HTTP services in sync with the current HTTP standards.
+
+## How to use this image
+
+This image only contains Apache httpd. To start a simple HTML server run:
+
+{%- set reference = ( image.reference | replace("-%RELEASE%", "") ) %}
+
+```ShellSession
+$ podman run -it --rm -p 8080:80 -v /path/to/html:/srv/www/htdocs/ {{reference}}
```

Comment:
You can use the recently added `image.pretty_reference` for this instead:

```suggestion
$ podman run -it --rm -p 8080:80 -v /path/to/html:/srv/www/htdocs/ {{image.pretty_reference}}
```

Reply:
I forgot that, but will fix it.